ush/forecast_predet.sh

```diff
@@ -781,5 +781,54 @@ GOCART_predet(){
   # FHMAX gets modified when IAU is on, so keep origianl value for GOCART output
   GOCART_MAX=${FHMAX}
 
+  #TODO: fix to copying data so that its required
+  ${NLN} "${COMIN_CHEM_INPUT}" "${DATA}/ChemInput"
+  # Create the ChemInput directory in the local run directory
+
+  # if [[ ! -d "${DATA}/ChemInput" ]]; then mkdir -p "${DATA}/ChemInput"; fi
```

Comment:
This is all just commented out for now? The expectation is that each file will be explicitly copied at a later date? Or should we do it now?

Reply:
I was having trouble with this part and so I did a $NLN to be able to move forward.  I'll look back at this again

Reply:
@arunchawla-NOAA could use some guidance on if we can use `${NLN}` or if we need to use `cpreq`

Reply:
@aerorahul could use some guidance if we need to use `cpreq` or if we can use ${NLN} to link the `COMIN_CHEM_INPUT` directory directly to the run directory similar to the way that the `ExtData` directory is linked to the run directory.